meson.build

```diff
@@ -1,7 +1,7 @@
 project('taisei', 'c',
     license : 'MIT',
     version : 'v1.4-dev',
-    meson_version : '>=0.48.0',
+    meson_version : '>=0.53.0',
```

Comment:
Since this has been bumped, we should also remove the `minimum_recommended_meson_version` code just below the `project` statement, and update the README.